src/styles/scss/Carousel.scss

```diff
@@ -0,0 +1,95 @@
+.carousel-container {
+  display: flex;
```

Comment:
Try to avoid using flex, rewriting it without it is a good learning experience. Also, flex is not well supported yet.